packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -63,6 +64,25 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return await this.provider.wasm.contractQuery(address, input, params)
   }
 
+  signAndSend = async (messages: MsgExecuteContract[]): Promise<TransactionResponse> => {
+    try {
+      const tx = await this.wallet.createAndSignTx({
+        msgs: messages,
+        ...(this.wallet.key instanceof LedgerKey && {
+          signMode: SignMode.SIGN_MODE_LEGACY_AMINO_JSON,
+        }),
+      })
+
+      const res = await this.provider.tx.broadcast(tx)
+
+      logger.debug(res)
```

Comment:
This debug log looks out of place, do we want it? Will this only appear with `--debug` flag?

Reply:
Yeah we don't really need it, removing it